gradle.properties

```diff
@@ -13,10 +13,10 @@ underCouchDownloadVersion=4.0.4
 researchgateReleaseVersion=2.8.0
 
 transportVersion=6.0.55
-stdlibRegexVersion=1.2.1
-stdlibTimeVersion=2.2.1
-stdlibIoVersion=1.2.1
-stdlibLogVersion=2.2.1
-stdlibOsVersion=1.2.1
+stdlibRegexVersion=1.3.0-20220425-094000-d073680
```

Comment:
Since some dependencies have been updated to their next minor versions, we need to bump the module to the next minor version.

Reply:
We already bumped the module to the next minor version to this release.